services/PermissionsService.ts

```diff
@@ -9,14 +9,18 @@ export default class PermissionsService {
   }
 
   public static canSeeEventAttendances(user: UserModel): boolean {
-    return user.isAdmin();
+    return user.isAdmin() || user.isMarketing();
   }
 
   public static canSubmitFeedback(user: UserModel): boolean {
     return user.state === UserState.ACTIVE;
   }
```

Comment:
btw, currently users stuck in the `PASSWORD_RESET` state can't submit feedback

```suggestion
  public static canSubmitFeedback(user: UserModel): boolean {
    return user.state === UserState.ACTIVE || user.state === UserState.PASSWORD_RESET;
  }
```